.github/workflows/multi-platform-test.yml

```diff
@@ -43,3 +43,5 @@ jobs:
       run: mvn clean test -Dprops.file=testconfig/ebean-sqlserver17.properties
     - name: mariadb
       run: mvn clean test -Dprops.file=testconfig/ebean-mariadb.properties
+    - name: db2
+      run: mvn clean test -Dprops.file=testconfig/ebean-db2.properties
```

Comment:
How can I run that workflow?

Reply:
Maybe it's a permissions thing ... let me see